requirements/default.txt

```diff
@@ -1,154 +1,205 @@
+    --hash=sha256:1cf5d84290c771eeecb734abe2c6c3120e9837eb12f99474141a862b9061ac51 \
+    --hash=sha256:0344b181e1aea37a58c218ccb0f0f771295de9aa25a625ed076e6996c6530f9e \
+    --hash=sha256:25250867a4cd1510fb755ef9cb38da3065def999d8e92c44e49a39b9b76bc893 \
+    --hash=sha256:317612d5d0ca4a9f7e42afb2add69b10be360784d21ce4ecfbca19f1f5eadf43 \
+    --hash=sha256:9d6266348b15b4a48623bf4d3e50445d8e581da413644f365805b321703d0fac \
+    --hash=sha256:ddca39cc55877653b5fcf59976d073e3d58c7c406ef54ae8e61ddf8782867182 \
+    --hash=sha256:988d2ec7560d42ef0ac34b3b97aad14c4f068792f00e1524fa1d3749fe4e4b64 \
+    --hash=sha256:7a9c6c62e6e05df5406e9b5235c31c376a22620ef26715a663cee57083b3c2ea \
+    --hash=sha256:7a75565181e75ba0b9fb174b58172bf6ea9b4331631cfe7bafff03f3641f5d73 \
+    --hash=sha256:94e4128ba1ea56f02522fffac65520091a9de3f5c00da31539e085e13db4771b \
+    --hash=sha256:92179bd68c2efe72924a99b6745a9172471931fc296f9bfdf9645b75eebd6344 \
+    --hash=sha256:b9358e203168fef7bfe9f430afaed3a2a624717a1d19c7afa7dfcbd76e3cd95c \
+    --hash=sha256:009e0bc09a57dbef4b601cb8b46a2abad51f5274c8be4bba276ff2884cd4cc53 \
+    --hash=sha256:d3ac07240e2304181ffdb13c099840b5eb555efc7be9344503c0c03aa681de79 \
+    --hash=sha256:40fa5630cd7d237cd93c4d4b64b9e5ed9273d1cfce55241c7f9066f5db70629d \
+    --hash=sha256:6c2f1a76a9ebd9ecf7825b9e20860139ca502c2bf1beabf6accf6c9e66a7e0c3 \
+    --hash=sha256:37f54452c7787dbdc0a634ca9773362b91709917f0b365ed14b831f03cbd34ba \
+    --hash=sha256:8f5942a4daf1ffac42109dc4a72f786af4baa4fa702ede1d7c57b4b696c2e7d6 \
+    --hash=sha256:bf708455cd1e9fa96c05126e89a0c59b200d086c7df7bbafc7d9be769e4149a3 \
+    --hash=sha256:82c40ea3ac1555e0462803380609fbe8b26f52620f3d4f8eb480cfd8ceed8a14 \
+    --hash=sha256:207ba4f9125a0a4200691e82d5eee7ea1485708eabe99a07fc7f08696fae62f4 \
+    --hash=sha256:0cd4c848f0e9d805d531e44973c8f48962e20eb7fc0edac3db4f9dbf9ed5ab82 \
+    --hash=sha256:57baf63aeb2965ca4b52613ce78e968b6d2bde700c97f6a7e8c6c236b51ab83e \
+    --hash=sha256:2954557393cfc9a5c11a5199c7a78cd9c0c793a047552d27b1636da50d013916 \
+    --hash=sha256:7c31dade89634807196a6b20ced831fbd5bec8a21c4e458ea950c9102c3aa96f \
+    --hash=sha256:1286dd16d0e46d59fa54582725986704a7a3f3d9aca6c5902a7eceb10c60cb7e \
+    --hash=sha256:697ff63bc5451e0b0db48ad205151123d25683b3754198be7ab5fcb44334e519 \
+    --hash=sha256:fc993c9331d91766d54757bbc70231e29d5ceb2d1ac08b1570feaa0c38ab9582 \
+    --hash=sha256:9d64fed2681552ed642e9c0cc831a9e95ab91de72b47d0cb68b5bf506ba88647 \
+    --hash=sha256:5c3213be557d0468f9df8fe2487eaf2990d9799202c5ff5cb8d394d09fad9b2a
+elasticsearch==5.4.0 \
+    --hash=sha256:8f02d626cff98befd298ffb1d19d8e519643d86d3aa27f6d7388566ec4ba48ee \
+    --hash=sha256:e754c688e20fe73160fb6f7f5b63f2a71c78788dc9e6908950681d3a39b56e85
+elasticsearch-dsl==5.3.0 \
+    --hash=sha256:de5fd68901c5b94e51e2ae4e8ad6f618b22665d89794d96f9b6b763986665527 \
+    --hash=sha256:de81b2322cde9bf1171c8a41d80596cdadd433b88b93daa09060af3a512523a7
```

Comment:
We talked about this yesterday, but figured I'd leave a comment here so we have it written down.

We can't update elasticsearch or elasticsearch-dsl--those need to get updated at the same time we update our Elasticsearch cluster.